buildSrc/src/main/kotlin/DocsLandingPage.kt

```diff
@@ -0,0 +1,53 @@
+import org.gradle.api.Project
```

Comment:
Take a pass on copyright headers. We may just want to grab the Ktlint rule from the Kotlin SDK.